src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ApiAttributesTest.php

```diff
@@ -84,12 +143,78 @@ public static function mapQueryStringProvider(): iterable
                 JSON,
             'expectedStatusCode' => 404,
         ];
+
+        yield 'empty query string mapping non-nullable attribute without default value' => [
+            'uri' => '/map-query-string-to-non-nullable-attribute-without-default-value.json',
+            'query' => [],
+            'expectedResponse' => <<<'JSON'
+                {
+                    "type": "https:\/\/symfony.com\/errors\/validation",
+                    "title": "Validation Failed",
+                    "status": 404,
+                    "detail": "filter: This value should be of type Symfony\\Bundle\\FrameworkBundle\\Tests\\Functional\\Filter.",
```

Comment:
This text can be changed depending of the version of the framework bundle. Using `Composer\InstalledVersions`, so that the test pass for all supported version.